packages/apps/transfer/src/hooks/use-did-update-effect.ts

```diff
@@ -1,6 +1,7 @@
 import { useEffect, useRef } from 'react';
 
-export default (fn: Function, inputs: any[]) => {
+// eslint-disable-next-line @typescript-eslint/ban-types
+export default (fn: Function, inputs: unknown[]): boolean => {
```

Comment:
We can type this easily as `() => void`.

```suggestion
export default (fn: () => void, inputs: unknown[]): boolean => {
```

Reply:
done